app/controllers/orgs/rosters_controller/lti_dependency.rb

```diff
@@ -52,11 +52,12 @@ def lms_membership
       membership_service = GitHubClassroom::LTI::MembershipService.new(
         membership_service_url,
         current_organization.lti_configuration.consumer_key,
-        current_organization.lti_configuration.shared_secret
+        current_organization.lti_configuration.shared_secret,
+        lti_version: current_organization.lti_configuration.lti_version
       )
 
       begin
-        membership_service.students
+        membership_service.students(body_params: current_organization.lti_configuration.context_membership_body_params)
```

Comment:
The changes to this file should have actually been made in #2174, but it's fine they weren't because they have default fallbacks. Better late than never I suppose 😅